### PR TITLE
remove: Add confirmation when trying to purge

### LINKED ISSUE #1228: remove -p: add a confirmation prompt
E.g. see `dvc/prompt.py`.

### PR DESCRIPTION
**Pull request by mroutis**
Close #1228

### CODE REVIEW

**Comment by shcheklein** [comment]
Looks good! Do we have tests for remove? I thinks it would be good to add a negative one.

**Comment by mroutis** [comment]
> Looks good! Do we have tests for remove? I thinks it would be good to add a negative one.

We do, @shcheklein , but there's no `--purge` one because testing the interaction is quite tiresome, but I can add one that test `remove` with `outs_only=False` (that's basically the same as doing --purge and input `yes` at the prompt)

**Comment by efiop** [comment]
@shcheklein  We don't have a framework for interactive stuff 🙁

@mroutis Please take a look at `dvc destroy` https://github.com/iterative/dvc/blob/master/dvc/command/destroy.py#L13 , where we raise an exception if user didn't confirm the action. I think we also should add `--force` option for `dvc remove`, to force `purging` without confirmation.

File: dvc/cli.py [hunk 1]
```diff
@@ -501,6 +501,12 @@ def parse_args(argv=None):
                         action='store_true',
                         default=False,
                         help='Remove DVC file and all its outputs.')
+    remove_parser.add_argument(
+                        '-f',
+                        '--force',
+                        action='store_true',
+                        default=False,
+                        help='Force destruction.')
```
Comment (efiop) [inline dvc/cli.py]:
```suggestion
                        help='Force purge.')
```

**Comment by efiop** [comment]
@mroutis Looks great! Thank you! One more thing. Could you please squash your commits(e.g. if you have N commits: `git rebase -i HEAD~N` and squash/reorganize + `git push -f`) _in the future_, so, if possible, every commit in the PR does something meaningful(e.g. no `fix line length`/`fix typo` and stuff like that, when it is not the main changes of the PR) and, again if possible, doesn't fail CI? This makes it much easier to debug in the future and also `git log` becomes much cleaner. In this particular PR, we can safely choose `Squash and merge` option, since there is only one logical part to it.

**Pull request merged**